src/components/Dashboard-Block.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Generic Dashboard wrapper block for displaying some information in the admin.
+ *
+ * @param {String} title   Title to display in collapsbile header.
+ * @param {Array} children React children.
+ */
+const DashboardBlock = ( { title, children } ) => {
+	return <div className="postbox-container">
+		<div className="postbox">
+			<button type="button" className="handlediv" aria-expanded="true">
+				<span className="screen-reader-text">Toggle panel: Activity</span>
```

Comment:
`Activity` should be `{ title }` I'm guessing

Reply:
Yup, good catch.